testsuite/smoke-test/src/indexer.rs

```diff
@@ -33,9 +33,24 @@ pub fn wipe_database(conn: &PgPoolConnection) {
     }
 }
 
+/// By default, skips test unless `INDEXER_DATABASE_URL` is set.
+/// In CI, will explode if `INDEXER_DATABASE_URL` is NOT set.
+pub fn should_skip() -> bool {
```

Comment:
can't we ... put this in a single place? Like import it from indexer::tailer::?

Reply:
I was trying to avoid exporting test-only code, but I can, sure :-)

Reply:
moved!